buildSrc/src/main/groovy/org/elasticsearch/gradle/SnippetsTask.groovy

```diff
@@ -110,7 +110,8 @@ public class SnippetsTask extends DefaultTask {
                     lastLanguageLine = lineNumber
                     return
                 }
-                if (line ==~ /\/\/ AUTOSENSE\s*/) {
+                if (line ==~ /\/\/ AUTOSENSE\s*/
+                        || line ==~ /\/\/ CONSOLE\s*/) {
```

Comment:
I think all the regexes suffer from this. Would you mind if I merge this now and open a new PR for them?